src/bin/wasm-tools/component.rs

```diff
@@ -107,6 +107,14 @@ pub struct NewOpts {
     /// Print the output in the WebAssembly text format instead of binary.
     #[clap(long, short = 't')]
     wat: bool,
+
+    /// Use memory.grow to realloc memory.
+    #[clap(long, short = 'r')]
+    realloc_via_memory_grow: bool,
+
+    /// Use memory.grow for stack allocation.
+    #[clap(long, short = 's')]
+    alloc_stack_via_memory_grow: bool,
```

Comment:
Also, is it worth having a separate knob for this than the one above? I'm not sure when the state would want to be allocated with `cabi_realloc` and the stack with `memory.grow` specifically, so the above knob may be enough perhaps?

Reply:
That works for me, removed